modal/mount.py

```diff
@@ -301,7 +301,8 @@ def add_local_dir(
         *,
         # Where the directory is placed within in the mount
         remote_path: Union[str, PurePosixPath, None] = None,
-        # Filter function for file selection; defaults to including all files
+        # Predicate filter function for file selection; function should return `True` for inclusion, `False` otherwise.
+        # Defaults to including all files.
```

Comment:
Could we clarify that the function is passed the filepath? Maybe that's obvious but we don't explicitly say it.